css/admin.css

```diff
@@ -91,10 +94,14 @@ Enqueued only within the admin screen. (Via enqueue.php)
hbmRsZSB7XG4gIGJhY2tncm91bmQtY29sb3I6ICNGOTk7IH1cbiAgLm1lbnUtaXRlbS1kZXB0aC03IC5tZW51LWl0ZW0tYmFyIC5tZW51LWl0ZW0taGFuZGxlIC5pcy1zdWJtZW51OjphZnRlciB7XG4gICAgY29udGVudDogJyAtIHRvbyBkZWVwISc7IH1cbiAgLm1lbnUtaXRlbS1kZXB0aC03IC5tZW51LWl0ZW0tYmFyIC5tZW51LWl0ZW0taGFuZGxlIC5pdGVtLXRpdGxlIHtcbiAgICBtYXJnaW4tcmlnaHQ6IDVlbTsgfVxuXG4ubWVudS1pdGVtLWRlcHRoLTggLm1lbnUtaXRlbS1iYXIgLm1lbnUtaXRlbS1oYW5kbGUge1xuICBiYWNrZ3JvdW5kLWNvbG9yOiAjRjk5OyB9XG4gIC5tZW51LWl0ZW0tZGVwdGgtOCAubWVudS1pdGVtLWJhciAubWVudS1pdGVtLWhhbmRsZSAuaXMtc3VibWVudTo6YWZ0ZXIge1xuICAgIGNvbnRlbnQ6ICcgLSB0b28gZGVlcCEnOyB9XG4gIC5tZW51LWl0ZW0tZGVwdGgtOCAubWVudS1pdGVtLWJhciAubWVudS1pdGVtLWhhbmRsZSAuaXRlbS10aXRsZSB7XG4gICAgbWFyZ2luLXJpZ2h0OiA1ZW07IH1cblxuLm1lbnUtaXRlbS1kZXB0aC05IC5tZW51LWl0ZW0tYmFyIC5tZW51LWl0ZW0taGFuZGxlIHtcbiAgYmFja2dyb3VuZC1jb2xvcjogI0Y5OTsgfVxuICAubWVudS1pdGVtLWRlcHRoLTkgLm1lbnUtaXRlbS1iYXIgLm1lbnUtaXRlbS1oYW5kbGUgLmlzLXN1Ym1lbnU6OmFmdGVyIHtcbiAgICBjb250ZW50OiAnIC0gdG9vIGRlZXAhJzsgfVxuICAubWVudS1pdGVtLWRlcHRoLTkgLm1lbnUtaXRlbS1iYXIgLm1lbnUtaXRlbS1oYW5kbGUgLml0ZW0tdGl0bGUge1xuICAgIG1hcmdpbi1yaWdodDogNWVtOyB9XG5cbi5tZW51LWl0ZW0tZGVwdGgtMTAgLm1lbnUtaXRlbS1iYXIgLm1lbnUtaXRlbS1oYW5kbGUge1xuICBiYWNrZ3JvdW5kLWNvbG9yOiAjRjk5OyB9XG4gIC5tZW51LWl0ZW0tZGVwdGgtMTAgLm1lbnUtaXRlbS1iYXIgLm1lbnUtaXRlbS1oYW5kbGUgLmlzLXN1Ym1lbnU6OmFmdGVyIHtcbiAgICBjb250ZW50OiAnIC0gdG9vIGRlZXAhJzsgfVxuICAubWVudS1pdGVtLWRlcHRoLTEwIC5tZW51LWl0ZW0tYmFyIC5tZW51LWl0ZW0taGFuZGxlIC5pdGVtLXRpdGxlIHtcbiAgICBtYXJnaW4tcmlnaHQ6IDVlbTsgfVxuXG4ubWVudS1pdGVtLWRlcHRoLTExIC5tZW51LWl0ZW0tYmFyIC5tZW51LWl0ZW0taGFuZGxlIHtcbiAgYmFja2dyb3VuZC1jb2xvcjogI0Y5OTsgfVxuICAubWVudS1pdGVtLWRlcHRoLTExIC5tZW51LWl0ZW0tYmFyIC5tZW51LWl0ZW0taGFuZGxlIC5pcy1zdWJtZW51OjphZnRlciB7XG4gICAgY29udGVudDogJyAtIHRvbyBkZWVwISc7IH1cbiAgLm1lbnUtaXRlbS1kZXB0aC0xMSAubWVudS1pdGVtLWJhciAubWVudS1pdGVtLWhhbmRsZSAuaXRlbS10aXRsZSB7XG4gICAgbWFyZ2luLXJpZ2h0OiA1ZW07IH1cblxuLm1lbnUtaXRlbS1kZXB0aC0xMiAubWVudS1pdGVtLWJhciAubWVudS1pdGVtLWhhbmRsZSB7XG4gIGJhY2tncm91bmQtY29sb3I6ICNGOTk7IH1cbiAgLm1lbnUtaXRlbS1kZXB0aC0xMiAubWVudS1pdGVtLWJhciAubWVudS1pdGVtLWhhbmRsZSAuaXMtc3VibWVudTo6YWZ0ZXIge1xuICAgIGNvbnRlbnQ6ICcgLSB0b28gZGVlcCEnOyB9XG4gIC5tZW51LWl0ZW0tZGVwdGgtMTIgLm1lbnUtaXRlbS1iYXIgLm1lbnUtaXRlbS1oYW5kbGUgLml0ZW0tdGl0bGUge1xuICAgIG1hcmdpbi1yaWdodDogNWVtOyB9XG5cbi5tZW51LWl0ZW0tZGVwdGgtMTMgLm1lbnUtaXRlbS1iYXIgLm1lbnUtaXRlbS1oYW5kbGUge1xuICBiYWNrZ3JvdW5kLWNvbG9yOiAjRjk5OyB9XG4gIC5tZW51LWl0ZW0tZGVwdGgtMTMgLm1lbnUtaXRlbS1iYXIgLm1lbnUtaXRlbS1oYW5kbGUgLmlzLXN1Ym1lbnU6OmFmdGVyIHtcbiAgICBjb250ZW50OiAnIC0gdG9vIGRlZXAhJzsgfVxuICAubWVudS1pdGVtLWRlcHRoLTEzIC5tZW51LWl0ZW0tYmFyIC5tZW51LWl0ZW0taGFuZGxlIC5pdGVtLXRpdGxlIHtcbiAgICBtYXJnaW4tcmlnaHQ6IDVlbTsgfVxuXG4vKi0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS1cbjIuIEZvbnQgQXdlc29tZSBmaXhlcyBmb3IgR3V0ZW5iZXJnXG4tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLSovXG4uZWRpdG9yLXN0eWxlcy13cmFwcGVyIHN2Zy5zdmctaW5saW5lLS1mYSB7XG4gIHZlcnRpY2FsLWFsaWduOiAtLjEyNXJlbTsgfVxuIl19 */
```

Comment:
It looks like this wasn't merged correctly, why do we still have **<<<<<<< HEAD** and **>>>>>>> develop**